### ISSUE TITLE
[Bug]: CN panic

### ISSUE THREAD

**Issue by zhangxu19830126** [matrixorigin/matrixone#10499]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93):
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior

panic: unexpected const expr isnull:true

goroutine 11534914 [running]:
github.com/matrixorigin/matrixone/pkg/vm/engine/disttae.getCompositeFilterFuncByExpr(0x0?, 0x80?)
        /opt/source/matrixone/pkg/vm/engine/disttae/util.go:1020 +0x60e
github.com/matrixorigin/matrixone/pkg/vm/engine/disttae.(*withFilterMixin).getCompositPKFilter(0xc06d5e69c0, 0x0?)
        /opt/source/matrixone/pkg/vm/engine/disttae/reader.go:168 +0x1eb
github.com/matrixorigin/matrixone/pkg/vm/engine/disttae.(*withFilterMixin).getReadFilter(0xc06d5e69c0?, 0xc0cd1c92d0?)
        /opt/source/matrixone/pkg/vm/engine/disttae/reader.go:139 +0x3b
github.com/matrixorigin/matrixone/pkg/vm/engine/disttae.(*blockReader).Read(0xc06d5e69c0, {0x45d836?, 0x2?}, {0xc0cd1c92d0, 0x7, 0x7}, 0x800000020?, 0x20000000c?, {0x2ed0828, 0xc0e2590f00})
        /opt/source/matrixone/pkg/vm/engine/disttae/reader.go:324 +0x10a
github.com/matrixorigin/matrixone/pkg/vm/engine/disttae.(*blockMergeReader).Read(0xc045b39270, {0x2ee2378, 0xc195598660}, {0xc0cd1c92d0, 0x7, 0x7}, 0x18?, 0xc0fa619fa0?, {0x2ed0828, 0xc0e2590f00})
        /opt/source/matrixone/pkg/vm/engine/disttae/reader.go:456 +0xc5
github.com/matrixorigin/matrixone/pkg/vm/engine/disttae.(*mergeReader).Read(0xc02f0fa1e0, {0x2ee2378, 0xc195598660}, {0xc0cd1c92d0, 0x7, 0x7}, 0x0?, 0x42f2001?, {0x2ed0828, 0xc0e2590f00})
        /opt/source/matrixone/pkg/vm/engine/disttae/reader.go:484 +0x10c
github.com/matrixorigin/matrixone/pkg/vm/engine/disttae.(*mergeReader).Read(0xc02f0fa1c8, {0x2ee2378, 0xc195598660}, {0xc0cd1c92d0, 0x7, 0x7}, 0x4abb00?, 0xc10a21b0f0?, {0x2ed0828, 0xc0e2590f00})
        /opt/source/matrixone/pkg/vm/engine/disttae/reader.go:484 +0x10c
github.com/matrixorigin/matrixone/pkg/vm/engine/disttae.(*mergeReader).Read(0xc02f0fa0f0, {0x2ee2378, 0xc195598660}, {0xc0cd1c92d0, 0x7, 0x7}, 0xc001748501?, 0x7f6510487448?, {0x2ed0828, 0xc0e2590f00})
        /opt/source/matrixone/pkg/vm/engine/disttae/reader.go:484 +0x10c
github.com/matrixorigin/matrixone/pkg/vm/pipeline.(*Pipeline).Run(0xc00f5719a8, {0x2ed0558, 0xc02f0fa0f0}, 0xc0e2590f00)
        /opt/source/matrixone/pkg/vm/pipeline/pipeline.go:79 +0x2bd
github.com/matrixorigin/matrixone/pkg/sql/compile.(*Scope).Run(0xc0e8951c70, 0xc045b393a0?)
        /opt/source/matrixone/pkg/sql/compile/scope.go:84 +0x16e
github.com/matrixorigin/matrixone/pkg/sql/compile.(*Scope).ParallelRun(0xc0e8951c70, 0xc031f38d80, 0x0)
        /opt/source/matrixone/pkg/sql/compile/scope.go:377 +0xc65
github.com/matrixorigin/matrixone/pkg/sql/compile.(*Scope).RemoteRun(0xc0e8951c70, 0xc031f38d80)
        /opt/source/matrixone/pkg/sql/compile/scope.go:188 +0xa5
github.com/matrixorigin/matrixone/pkg/sql/compile.(*Scope).MergeRun.func3(0x0?)
        /opt/source/matrixone/pkg/sql/compile/scope.go:122 +0x2f
created by github.com/matrixorigin/matrixone/pkg/sql/compile.(*Scope).MergeRun
        /opt/source/matrixone/pkg/sql/compile/scope.go:121 +0x1f8

### Steps to Reproduce

```Markdown
run tpcc in pessimistic mode.
```

**Comment by triump2020** [comment]
on vacation

**Comment by triump2020** [comment]
@zhangxu19830126   Had fixed by #10673
